client/state/analytics/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const continuousTracking = createReducer( 'CHECKING', {
+	[ ANALYTICS_CONTINOUS_MONITOR_ON ]: () => 'TRACKING',
+	[ ANALYTICS_CONTINOUS_MONITOR_OFF ]: () => 'NOT_TRACKING'
+} );
```

Comment:
Since there are only two conceivable possibilities for state here -- tracking, or not tracking -- let's use a bool.
When we first start Calypso, we won't be tracking, so the initial value should be `false`:

```js
export const continuousTracking = createReducer( false, {
	[ ANALYTICS_CONTINOUS_MONITOR_ON ]: () => true,
	[ ANALYTICS_CONTINOUS_MONITOR_OFF ]: () => false
} );
```

If we aren't ever turning it off, we might consider dropping `ANALYTICS_CONTINOUS_MONITOR_OFF` (at least for now), b/c YAGNI.